examples/learning/helloworld64.html

```diff
@@ -40,36 +40,35 @@ <h1>'Hello, world!' example</h1>
   // Opening PDF by passing its binary data as a string. It is still preferable
   // to use Uint8Array, but string or array-like structure will work too.
   var loadingTask = pdfjsLib.getDocument({ data: pdfData, });
-  loadingTask.promise.then(function(pdf) {
+  (async function () {
+    var pdf = await loadingTask;
```

Comment:
Sorry, but this doesn't actually work!
The `loadingTask` is *not* a Promise, but rather a structure with a `promise`-property; see https://github.com/mozilla/pdf.js/blob/f6d9d91965a97c80b6f5b335a27cdb4d36b1b9e2/src/display/api.js#L527-L601

Hence the *correct* change would be `const pdf = await loadingTask.promise;`.

Furthermore, given that the example cannot possibly work as-is: please make sure to (in this case manually) test the changes that you make to ensure that your code actually works.
Testing this example locally, after making changes, includes:
 - Running `gulp dist-install`.
 - Running `gulp server`.
 - Opening http://localhost:8888/examples/learning/helloworld64.html in a browser and check that the example loads and works correctly without e.g. any errors printed in the console.